lte/gateway/Makefile

```diff
@@ -72,6 +72,8 @@ all: build
 
 build: build_python build_common build_oai build_sctpd build_session_manager build_connection_tracker build_envoy_controller build_li_agent ## Build all
 
+amf_build: build_common build_oai build_sctpd build_session_manager build_connection_tracker build_envoy_controller build_li_agent ## Build all
```

Comment:
why is there an `amf_build`???

Reply:
it was merged unintentionally, removed amf_build.